### PR TITLE
Test for azure-pipelines

### PR DESCRIPTION
**Pull request by mariusvniekerk**

### CODE REVIEW

**Comment by mariusvniekerk** [comment]
@scopatz seems the checks aren't showing up?

**Comment by scopatz** [comment]
There they are!

**Comment by mariusvniekerk** [comment]
look, status thingies!

File: .azure-pipelines/azure-pipelines-osx.yml [hunk 1]
```diff
@@ -0,0 +1,82 @@
+jobs:
+- job: osx_64
+  pool:
+    vmImage: xcode9-macos10.13
```
Comment (minrk) [inline .azure-pipelines/azure-pipelines-osx.yml]: This is a quite recent xcode. I *think* with properly setting CONDA_BUILD_SYSROOT this should be okay (good, even), but updating this version should be considered deliberately.

**Pull request closed without merging**